package.json

```diff
@@ -895,6 +905,11 @@
           "when": "viewItem =~ /^gistFile/",
           "group": "base@3"
         },
+        {
+          "command": "gistpad.viewForkedGist",
+          "when": "viewItem =~ /^gists.gist/",
+          "group": "fork@1"
```

Comment:
Maybe we could add this to the `star` group? That way we don't add another group. We could make this `star@1`, and the star/unstar commands `star@2`.

Reply:
Sounds good.